multiworld/envs/mujoco/__init__.py

```diff
@@ -18,328 +18,214 @@ def register_custom_envs():
     """
     Reaching tasks
     """
+
     register(
         id='SawyerReachXYEnv-v0',
         entry_point='multiworld.envs.mujoco.sawyer_xyz.sawyer_reach:SawyerReachXYEnv',
         tags={
-            'git-commit-hash': 'c5e15f7',
-            'author': 'vitchyr'
+            'git-commit-hash': '2d95c75',
+            'author': 'murtaza'
         },
         kwargs={
-            'hide_goal_markers': False,
+            'hide_goal_markers': True,
+            'norm_order':2,
         },
     )
+
     register(
         id='Image48SawyerReachXYEnv-v0',
         entry_point=create_image_48_sawyer_reach_xy_env_v0,
         tags={
-            'git-commit-hash': 'c5e15f7',
+            'git-commit-hash': '2d95c75',
             'author': 'vitchyr'
         },
     )
     register(
         id='Image84SawyerReachXYEnv-v0',
         entry_point=create_image_84_sawyer_reach_xy_env_v0,
         tags={
-            'git-commit-hash': 'c5e15f7',
+            'git-commit-hash': '2d95c75',
```

Comment:
I don't think we should just replace the git commit hash. Seems like you're making a new version of the environments, so then we should just make a new version in `__init__.py`. We shouldn't be afraid of having a large `__init__.py` file.